spec/feature/book_show_has_reviews_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+describe "when a user visits a book's show page" do
+  it "they see the book's title" do
+		book = Book.create(title: "The Brothers Karamazov")
+		visit "/books/#{book.id}" 
+
+		expect(page).to have_content("The Brothers Karamazov")
+	end 
+
+	it "they also see a list of reviews" do
+		book = Book.create(title: "The Brothers Karamazov")
+		review1 = Review.create(body: "People are really dramatic", rating: 5)
+
```

Comment:
a user should also be created and used in the review. 

```ruby 
user = User.create(name: "Ilana")
review1 = Review.create(body: "do stuff", rating: 4, user: user, book: book)
```

doesnt look like that relationship was created, even with a book and a review...would want to attach this review to a specific book and user.